packages/backend/src/docs/rest-api.yml

```
@@ -119,18 +121,263 @@ paths:
+      description: "Gets the discord channel id from the database"
+      operationId: "getDiscordChannelRequest"
+      security:
+        - Bearer: []
+      produces:
+      - "application/json"
+      responses:
+        "200":
+          description: "Successful operation"
+          schema:
+            $ref: "#/definitions/discordChannelRequest"
+        "400":
+          description: "Invalid request body"
+          schema:
+            $ref: "#/definitions/ApiError"
+        "401":
+          description: "Invalid token!"
+          schema: 
+            $ref: "#/definitions/ApiError"
+        "500":
+          description: "Internal server error"
+          schema:
+            $ref: "#/definitions/ApiError"
+        "503":
+          description: "Service Unavailable"
+          schema: 
+            $ref: "#/definitions/ApiError"
+    put:
+      tags:
+      - "Discord channel id"
+      summary: "Sets discord channel id"
+      description: "Sets the discord channel id on the database"
+      operationId: "setDiscordChannelRequest"
+      security:
+        - Bearer: []
+      consumes:
+      - "application/json"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "body"
+        in: "body"
+        description: "New discord channel id"
+        required: true
+        schema:
+         $ref: "#/definitions/discordChannelRequest"
+      responses:
+        "200":
+          description: "Successful operation"
+        "400":
+          description: "Invalid request body"
+          schema:
+            $ref: "#/definitions/ApiError"
+        "401":
+          description: "Invalid token!"
+          schema: 
+            $ref: "#/definitions/ApiError"
+        "500":
+          description: "Internal server error"
+          schema:
+            $ref: "#/definitions/ApiError"
+  /settings/courses:
+    get:
+      tags:
+      - "Course"
+      summary: "Gets course list"
+      description: "Gets the course list from the database/moodle"
+      operationId: "getCourseListRequest"
+      security:
+        - Bearer: []
+      produces:
+      - "application/json"
+      responses:
+        "200":
+          description: "Successful operation"
+          schema:
+            $ref: "#/definitions/getCourseListRequest"
+        "400":
+          description: "Invalid request body"
+          schema:
+            $ref: "#/definitions/ApiError"
+        "401":
+          description: "Invalid token!"
+          schema: 
+            $ref: "#/definitions/ApiError"
+        "404":
+          description: "No courses found"
+          schema:
+            $ref: "#/definitions/ApiError"
+        "500":
+          description: "Internal server error"
+          schema:
+            $ref: "#/definitions/ApiError"
+  /settings/courses/{id}:
+    put:
+      tags:
+      - "Course"
+      summary: "Sets single course status"
+      description: "Sets a single course status to active/inactive (for notifications)"
```

Wenn ich versuche, das als "Leie" zu verstehen, wirkt der Satz sehr zerpflügt. Anbei mein Vorschlag 😄 
```suggestion
      description: "Enables or disables the notifications assigned to the course id. "
```